src/app/components/match/match.component.html

```diff
@@ -1,115 +1,178 @@
-            <td>{{request.numberOfOccurrences}}</td>
-            <td>{{request.region}}</td>
-          </tr>
-          </tbody>
-        </table>
       </div>
-
-        <div class="input-group mb-1">
-          <h4 class="secondary-header mt-2 mb-0">
-            בחירת הצעת התנדבות
-          </h4>
-          <table class="table rtl table-sm table-hover" dir="rtl">
-            <thead class="thead">
-            <tr>
-              <th scope="col">שם</th>
-              <th scope="col">מספר מתנדבים</th>
-              <th scope="col">איש קשר</th>
-              <th scope="col">ימות השבוע</th>
-              <th scope="col">איזורים</th>
-            </tr>
+    </div>
+    <div class="row">
+        <div class="col-5">
+            <h4 class="secondary-header mt-2 mb-0">
+                בחירת בקשת התנדבות
+            </h4>
+        </div>
+        <div class="col-2"></div>
+        <div class="col-5">
+            <h4 class="secondary-header mt-2 mb-0">
+                בחירת הצעת התנדבות
+            </h4>
+        </div>
+    </div>
+    <div class="row mb-1">
+        <div class="col-5">
+            <ul id="accordion" class="list-group p-0 mx-0">
+                <li *ngFor="let request of volunteeringRequests; index as i" class="list-group-item align-items-start p-0 m-0">
+                    <div class="card p-0 m-0">
+                        <div (click)="onSelectRequest(request)" [ngClass]="{'is-selected': selectedRequest===request}"
+                            class="row card-header py-0 mx-0" aria-expanded="true" [attr.aria-controls]="'#collapse'+i" [attr.id]="'headingO'+i">
+                           <div class="col-1 d-flex align-items-center justify-content-center">
+                                <i class="fas fa-plus-circle fa-2x pl-1" data-toggle="collapse" [attr.data-target]="'#collapse'+i"></i>
+                            </div>                            
+                            <div class="col-11">
+                                <div class="row"><h5>{{request.title}}</h5></div>
+                                <div class="row"><div class="card-subtitle">עבור {{request.organization}}</div></div>
+                            </div>
+                        </div>                            
+                        <div [attr.id]="'collapse'+i" class="collapse" [attr.aria-labelledby]="'headingO'+i">
+                            <div class="card-body m-1 py-0">
+                                <div class="mb-1 d-flex justify-content-between">
+                                    <h6 class="mb-1">{{request.content.title}}</h6>
+                                    <div>רב פעמי: <i class="far fa-check-square" *ngIf="request.multiOccurrence"></i>
+                                                <i class="far fa-square" *ngIf="!request.multiOccurrence"></i>
+                                    </div>
+                                </div>
+                                <p class="mb-1">{{request.about}}</p>
```

Comment:
Is `request.about` limited in size? Maybe a length-limiting pipe should be added here (even if it is).

Reply:
Why use limiting pipe in a read only property?

Reply:
To prevent too-long text from breaking the display